static/app/views/issueList/addViewPage.tsx

```diff
@@ -97,18 +98,55 @@ function SearchSuggestionList({
   searchSuggestions,
   type,
 }: SearchSuggestionListProps) {
-  const {onNewViewSaved} = useContext(NewTabContext);
+  const {onNewViewsSaved} = useContext(NewTabContext);
   const organization = useOrganization();
 
   return (
     <Suggestions>
-      <TitleWrapper>{title}</TitleWrapper>
+      <TitleWrapper>
+        {title}
+        {type === 'saved_searches' && (
+          <StyledButton
+            size="zero"
+            onClick={e => {
+              e.stopPropagation();
+              openConfirmModal({
+                message: (
+                  <b style={{display: 'flex', justifyContent: 'center'}}>
+                    {t('Save ') +
```

Comment:
`tn` is the right util to use here - you want translatable strings to be unbroken, otherwise the translations will be off. It also handles plurals